mlrun/api/utils/clients/iguazio.py

```diff
@@ -505,11 +503,35 @@ def _generate_auth_info_from_session_verification_response(
         if x_unix_uid and x_unix_uid.lower() != "unknown":
             user_unix_id = int(x_unix_uid)
 
+        # get user and group ids from response body
```

Comment:
Maybe instead of this comment you can create a function which it purpose it to resolve params which may have responses in both body and headers.